src/components/menu-top/LoadShow.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="modal-card">
+    <header class="modal-card-head">
+      <p class="modal-card-title">
+        Load Show
+      </p>
+    </header>
+
+    <section class="modal-card-body">
+      <b-field class="file">
+        <b-upload
+          v-model="file"
+          accept=".shw"
+          @input="loadShow"
+        >
+          <a class="button is-primary">
+            <b-icon icon="upload" />
+            <span>Click to load</span>
+          </a>
+        </b-upload>
+        <span
+          v-if="file"
+          class="file-name"
+        >
+          {{ file.name }}
+        </span>
+      </b-field>
+      <b-message
+        v-if="parseError"
+        title="Error"
+        type="is-danger"
+      >
+        {{ parseError }}
+      </b-message>
+      <b-field v-if="show">
+        Number of marchers: {{ numMarchers }}
+      </b-field>
+      <b-field v-if="show">
```

Comment:
I think the `<b-field v-if="show">` are causing errors. When I click File -> Load, I am getting the following errors:
![image](https://user-images.githubusercontent.com/29964426/72099612-d0291e80-32d5-11ea-934b-b86c2d434725.png)

Reply:
Yep, this should be showPreview.  Thanks for catching.  Fixing in the next update.